internal/ioutil/print.go

```diff
@@ -1,25 +1,32 @@
 package ioutil
 
 import (
+	"context"
 	"fmt"
 	"io"
 	"text/tabwriter"
 
+	"github.com/notaryproject/notation-go"
 	"github.com/notaryproject/notation-go/config"
-	"github.com/notaryproject/notation-go/plugin/manager"
-	"github.com/notaryproject/notation-go/verification"
+	"github.com/notaryproject/notation-go/plugin"
+	"github.com/notaryproject/notation-go/plugin/proto"
 )
 
 func newTabWriter(w io.Writer) *tabwriter.Writer {
 	return tabwriter.NewWriter(w, 0, 0, 3, ' ', 0)
 }
 
-func PrintPlugins(w io.Writer, v []*manager.Plugin) error {
+func PrintPlugins(ctx context.Context, w io.Writer, v []plugin.Plugin) error {
 	tw := newTabWriter(w)
 	fmt.Fprintln(tw, "NAME\tDESCRIPTION\tVERSION\tCAPABILITIES\tERROR\t")
 	for _, p := range v {
-		fmt.Fprintf(tw, "%s\t%s\t%s\t%v\t%v\t\n",
-			p.Name, p.Description, p.Version, p.Capabilities, p.Err)
+		req := &proto.GetMetadataRequest{}
+		metadata, err := p.GetMetadata(ctx, req)
+		if err != nil {
+			return err
+		}
+		fmt.Fprintf(tw, "%s\t%s\t%v\t%v\t\n",
+			metadata.Name, metadata.Description, metadata.Version, metadata.Capabilities)
```

Comment:
@yizha1 The notation plugin list command doesn't show error information now, which is inconsistent with spec. Is it ok?

Reply:
@yizha1 Added it back.